design/editor/editor-ideas.md

```diff
@@ -238,7 +238,11 @@ e.g. you have a test `calculate_sum_test` that only uses the function `add`, whe
 - In the file explorer, auto-close subtree after unused for certain time.
 - Ability to right click error message > "create github issue".
 - Record (locally) all steps performed by editor + logs and make them searchable for user. The user should be able to scrub through these steps.
-
+- clustering of search results (rg like) based on statistics of result; amount of spaces, avg word length, amount of numbers...
+- Search engine to answer questions about your code base, logs, all intermediary values. Examples: search for "xy" in a string, search for xy in a variable name, show all intermediary values of index variable...
+- When a stacktrace is shown in the logs, highlight all lines of code in the editor that are in the stacktrace.
+- Double-click to select all child expressions of parent: [example](https://twitter.com/disconcision/status/1587156531203678208?s=20&t=GySrwPVnMB6rDKFqRuUcBw).
+- We should encourage users to ask editor something in the command window; e.g. "hide all types", "List docs", "switch light mode"... If the questions can not be matched to existing actions, the user should be given the option to share the question with editor developers so the action can be created or mapped to an existing action.
```

Comment:
Not a blocker, but I prefer if we say things like "could" over "should" when we're still in the idea stage. 😄

Reply:
:+1: I'll keep it in mind